github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1475,3 +1475,67 @@ periodics:
         resources:
           requests:
             memory: "29Gi"
+- annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cluster: prow-workloads
+  cron: 40 7,15,23 * * *
```

Comment:
I think we should shift the schedule a bit, there is another periodic lane that starts at the same times.

Reply:
Changed it to run once a day at 6am.